Sources/PaymentsCore/api/EligibilityAPI.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+public class EligibilityAPI {
```

Comment:
👋 Hey @kgangineni - I am catching up on some of the PRs since I've been away.

Question for you - can you provide some background on what the purpose of this the EligilibityAPI is?

I see that it is public, which means it is meant for merchant use - when would a merchant use this?

Reply:
hey..
this is for merchants to check if they have access to payment source type like paypal, venmo etc as check point before showing it to user

Reply:
Gotcha. Do we have a ticket for adding docstrings to this public class?

Also curious - was this feature a request from product/PMs?

Reply:
good callout, need to decide on it, weahter it will be public API or we will be used by us while showing payment methods, either way will need at ticket for this

Reply:
👋 I made a JIRA for this work `DTNOR-578`